cypress/fixtures/successful_signup_of_university_response.json

```diff
@@ -2,8 +2,8 @@
   "status": "success",
   "data": {
     "allow_password_change": false,
-    "name": "Harvard",
-    "role": "University",
+    "name": "harvard",
```

Comment:
Is the name really meant to be in lower case?

Reply:
Fixed.